crates/swc_ecma_minifier/src/compress/optimize/strings.rs

```diff
@@ -137,4 +137,29 @@ impl Optimizer<'_> {
             _ => {}
         }
     }
+
+    pub(super) fn reduce_escaped_newline_for_str_lit(&mut self, expr: &mut Expr) {
```

Comment:
Do we still need this, even with the codegen config?

Reply:
This is used to transform string literal to template literal. Maybe should rename the method name, it's not clear enough.

Reply:
Document added.